.github/dependabot.yml

```diff
@@ -32,3 +32,12 @@ updates:
         update-types:
           - "version-update:semver-patch"
           - "version-update:semver-minor"
+  - package-ecosystem: terraform
```

Comment:
This shouldn't need to be here for test modules - I know this is a result of the pathspec change below